src/votekit/ballot_generator.py

```diff
@@ -183,17 +198,56 @@ def _preference_interval(
 
 
 class BallotSimplex(BallotGenerator):
-    def __init__(self, alpha: float, **data):
-        self.alpha = alpha
-        super().__init__(**data)
+    def __init__(
+        self, alpha: Optional[float] = None, point: Optional[dict] = None, **data
+    ):
+        """
+        Base class for ballot generation with a ballot simplex
+        Args:
+            alpha (float, optional): alpha parameter for ballot simplex. Defaults to None.
+            point (dict, optional): a point in the ballot simplex,
+            with candidate as keys and electoral support as values. Defaults to None.
+        """
+        if alpha or point:
+            self.alpha = alpha
+            self.point = point
+            super().__init__(**data)
+
+    @classmethod
+    def from_point(cls, point: dict, **data):
+        if sum(point.values()) != 1.0:
+            raise ValueError(
+                f"probability distribution from point ({point.values()}) does not sum to 1"
+            )
+        return cls(point=point, **data)
+
+    @classmethod
+    def from_alpha(cls, alpha: float, **data):
+        return cls(alpha=alpha, **data)
 
     def generate_profile(self, number_of_ballots) -> PreferenceProfile:
         perm_set = it.permutations(self.candidates, self.ballot_length)
-        # Create a list of every perm [['A', 'B', 'C'], ['A', 'C', 'B'], ...]
+
         perm_rankings = [list(value) for value in perm_set]
 
-        # IAC Process is equivalent to drawing from dirichlet dist with uniform parameters
-        draw_probabilities = np.random.dirichlet([self.alpha] * len(perm_rankings))
+        if self.alpha:
+            draw_probabilities = list(
+                np.random.dirichlet([self.alpha] * len(perm_rankings))
+            )
+        elif self.point:
+            draw_probabilities = [
```

Comment:
this code gets surprisingly complex w/ nested list comprehensions, reduce, and conditionals in a lambda.  I'd add a comment to help walk readers through it